omegatrade-app/backend/app/models/user.model.js

```diff
@@ -0,0 +1,37 @@
+'user strict';
+const database = require('./../config/database.js');
+var User = function () {};
+
+User.registerUser = async function (user, cb) {
+  const usersTable = database.table('users');
+  try {
+    await usersTable.insert(user);
+    cb(null, true);
+    return true;
+  } catch (err) {
+    cb(err, null);
+      return;
+  } 
+}
+
+User.findUser = async function (email, cb) {
+  try {
+    const query = {
+      sql: 'SELECT userId,fullName,businessEmail,password,photoUrl,provider FROM users where businessEmail = @businessEmail',
+      params: {
+        businessEmail: email
+      }
+    };
+    let result = await database.run(query);
```

Comment:
nit: prefer `const` over `let` whenever possible.

Reply:
`result` variable changed to const.